demos/imagenet_classification_demo/README.md

```diff
@@ -0,0 +1,120 @@
+
+You can stop the demo by pressing "Esc" button. After that, the average metrics values would be printed to the console.
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
+
+## Running
+
+Running the application with the <code>-h</code> option yields the following usage message:
+```sh
+./imagenet_classification_demo -h
+
+imagenet_classification_demo [OPTION]
+Options:
+
+    -h                        Print a usage message.
+    -i "<path>"               Required. Path to a folder with images or path to an image file.
+    -m "<path>"               Required. Path to an .xml file with a trained model.
+      -l "<absolute_path>"    Required for CPU custom layers.Absolute path to a shared library with the kernels implementation.
+          Or
+      -c "<absolute_path>"    Required for GPU custom kernels. Absolute path to the .xml file with kernels description.
+    -labels "<path>"          Required. Path to .txt file with imagenet labels.
+    -gt "<path>"              Optional. Path to ground truth .txt file.
+    -d "<device>"             Optional. Specify the target device to infer on (the list of available devices is shown below). Default value is CPU. Sample will look for a suitable plugin for device specified.
+    -b "<integer>"            Optional. Specify batch to infer. Default value is 1.
+    -nthreads "<integer>"     Optional. Specify count of threads.
+    -nstreams "<integer>"     Optional. Specify count of streams.
+    -nireq "<integer>"        Optional. Number of infer requests.
+    -nt "<integer>"           Optional. Number of top results. Default value is 5. Must be >= 1.
+    -res "<WxH>"              Optional. Set image grid resolution in format WxH. Default value is 1920x1080.
+    -no_show                  Optional. Disable showing of processed images.
+    -time "<integer>"         Optional. Time in seconds to execute program. Default is -1 (infinite time).
+    -u                        Optional. List of monitors to show initially.
+```
+
+The number of InferRequests is specified by -nireq flag.
+Each InferRequest acts as a "buffer": it waits in queue before being filled with images and sent for inference, then after
+the inference completes, it waits in queue before its results would be processed.
+Increasing the number of InferRequests usually increases performance, because in that case multiple InferRequests can be
+processed simultaneously if the device supports parallelization.
```

Comment:
This way of line breaking looks strange. Looks that GitHub's preview width is slightly less than 120 symbols.